HLM_PV_Import/pv_import.py

```diff
@@ -0,0 +1,91 @@
+from HLM_PV_Import.ca_wrapper import PvMonitors
+from HLM_PV_Import.user_config import UserConfig
+from HLM_PV_Import.db_functions import add_measurement
+from HLM_PV_Import.settings import PvImportConfig
+from HLM_PV_Import.logger import log_error
+from HLM_PV_Import.settings import CA
+from collections import defaultdict
+import time
+
+LOOP_TIMER = PvImportConfig.LOOP_TIMER   # The timer between each PV import loop
+
+
+class PvImport:
+    def __init__(self, pv_monitors: PvMonitors, user_config: UserConfig):
+        self.pv_monitors = pv_monitors
+        self.config = user_config
+        self.tasks = {}
+        self.running = False
+
+    def set_up(self):
+        """
+        Requirements and checks before importing PV data.
+        """
+        # Check if the user configuration is valid
+        self.config.run_checks()
+
+        # Initialize tasks
+        for record in self.config.records:
+            self.tasks[record] = 0
+
+    def start(self):
+        """
+        Starts the PV data importing loop.
+        """
+        self.running = True  # in case it was previously stopped
+        start_time = time.time()
+
+        while self.running:
+            time.sleep(LOOP_TIMER - ((time.time() - start_time) % LOOP_TIMER))
```

Comment:
**should**: Do you really need this level of accuracy? I think if you do you should add a brief comment here to say what you're doing